scalameta/parsers/js/src/main/scala/scala/meta/parsers/JSFacade.scala

```diff
@@ -108,13 +108,11 @@ object JSFacade {
         }
     }
 
-  @JSExportTopLevel("default")
```

Comment:
should it be done via a JSExportTopLevel on the containing object, along with a JSExport the methods?

https://www.scala-js.org/doc/interoperability/export-to-javascript.html

Reply:
I think we want "top level" exports, i.e. it should result in:

```js
module.exports = {
  parseSource,
  parseStat
}
```

Reply:
specifically, we fall into the "Exporting top-level methods" case from the docs